nixnet/database/_signal.py

```diff
@@ -13,8 +13,9 @@ class Signal(object):
     """Database signal"""
 
     def __init__(self, handle):
+        # type: (int) -> None
         self._handle = handle
-        self._dbc_attributes = None
+        self._dbc_attributes = None  # type: typing.Union[None, _dbc_attributes.DbcAttributeCollection]
```

Comment:
You can use `Optional` instead

https://docs.python.org/3/library/typing.html#typing.Optional

Reply:
I like that more. I'll do that.